app/src/components/new-request/stats.vue

```diff
@@ -109,13 +109,33 @@ export default class Stats extends Vue {
 
   @Action(useStore) setStats!: ActionBindingIF
 
+  /**
+   * Normalize a wait time value to a finite number.
+   *
+   * Semantics:
+   * - waitTime > 0: explicit positive wait time returned. Otherwise, means disabled.
+   */
+  normalizeWaitTime (waitingTime: any): number {
+    const num = Number(waitingTime)
+    if (Number.isFinite(num)) {
+      return num
+    }
+    return 0
+  }
+
   async created (): Promise<void> {
     if (
+      // FF returns 0 either it sets to 0 or the FF is disabled
```

Comment:
Can you please explain what the different FF values do? And what should be displayed in each case?